ossdbtoolsservice/driver/types/mysql_driver.py

```diff
@@ -95,21 +100,20 @@ def __init__(self, conn_params: {}, config: Optional[Configuration] = None):
         if 'port' not in self._connection_options or not self._connection_options['port']:
             self._connection_options['port'] = constants.DEFAULT_PORT[constants.MYSQL_PROVIDER_NAME]
 
-        self._set_ssl_options(conn_params)
-
         # Setting autocommit to True initally
         self._autocommit_status = True
```

Comment:
If we are not setting usePure explicitly, 
Can we add a log/metric to capture fallbacks?

Reply:
Added check and logging for which extension the connector is using.